lib/index.js

```diff
@@ -71,3 +71,30 @@ export function toPosition(range) {
     end: toPoint(range.end)
   }
 }
+
+/**
+ * Convert a unist point, position, or undefined to an LSP range.
+ *
+ * @param {Readonly<UnistPosition> | Readonly<Point> | undefined} place
+ *   The unist point or position to convert.
+ *   If place is undefined, this returns an empty range at the beginning of the document.
+ * @returns {Range}
+ *   The LSP range.
+ */
+export function fromPlace(place) {
```

Comment:
The other functions don’t support `undefined`. I don‘t think `undefined` should be a “place”. So I’d say either add `undefined` support everywhere or remove it here

Reply:
The idea is to support everything supported by a [vfile message place](https://github.com/vfile/vfile-message#fields), which includes undefined. In that sense `point` and `position` are unist terms, whereas `place` is more of a vfile term.

Reply:
`node.position` can be missing too, but we don’t support that here?